entrypoint.sh

```diff
@@ -4,14 +4,17 @@ set -e
 # If we are called with default arguments, invoke SickGear with datadir and
 # setup the environment
 if [ "$1" = 'SickBeard.py' ]; then
+  APP_DATA_UIDGID="${APP_UID:=0}:${APP_GID:=0}"
   if [ ! -f ${APP_DATA}/config.ini ]; then
-    cp /template/config.ini ${APP_DATA}/
+    exec gosu ${APP_DATA_UIDGID} cp /template/config.ini ${APP_DATA}/
```

Comment:
ok, i asssumed you had already tested and reached point of completion, so i have just merged, now a fix PR is required